man/sd_bus_message_new_method_error.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version='1.0'?>
+<!DOCTYPE refentry PUBLIC "-//OASIS//DTD DocBook XML V4.2//EN"
+"http://www.oasis-open.org/docbook/xml/4.2/docbookx.dtd">
+
+<!--
+  SPDX-License-Identifier: LGPL-2.1+
+-->
+
+<refentry id="sd_bus_message_new_method_error"
+          xmlns:xi="http://www.w3.org/2001/XInclude">
+
+  <refentryinfo>
+    <title>sd_bus_message_new_method_error</title>
+    <productname>systemd</productname>
+  </refentryinfo>
+
+  <refmeta>
+    <refentrytitle>sd_bus_message_new_method_error</refentrytitle>
+    <manvolnum>3</manvolnum>
+  </refmeta>
+
+  <refnamediv>
+    <refname>sd_bus_message_new_method_error</refname>
+    <refname>sd_bus_message_new_method_errorf</refname>
+    <refname>sd_bus_message_new_method_errno</refname>
+    <refname>sd_bus_message_new_method_errnof</refname>
+
+    <refpurpose>Create a an error reply for a method call</refpurpose>
+  </refnamediv>
+
+  <refsynopsisdiv>
+    <funcsynopsis>
+      <funcsynopsisinfo>#include &lt;systemd/sd-bus.h&gt;</funcsynopsisinfo>
+
+      <funcprototype>
+        <funcdef>int sd_bus_message_new_method_error</funcdef>
+        <paramdef>sd_bus_message *<parameter>call</parameter></paramdef>
+        <paramdef>sd_bus_message **<parameter>m</parameter></paramdef>
+        <paramdef>const sd_bus_error *<parameter>e</parameter></paramdef>
+      </funcprototype>
+
+      <funcprototype>
+        <funcdef>int sd_bus_message_new_method_errorf</funcdef>
+        <paramdef>sd_bus_message *<parameter>call</parameter></paramdef>
+        <paramdef>sd_bus_message **<parameter>m</parameter></paramdef>
+        <paramdef>const char *<parameter>name</parameter></paramdef>
+        <paramdef>const char *<parameter>format</parameter></paramdef>
+        <paramdef>...</paramdef>
```

Comment:
not that it matters, and not really about this specific patch, but i wonder if it wouldn't be nicer to use unicode `…` here, instead of three `.`

It doesn't make much difference for the man page (as the docbook style relaces `…` → `...` anyway, but it might be nicer for the typographer in me and it will appear like that in the html version of the docs...

Reply:
There are cases, like bootup(7) where the width matters, and having the unicode char which is then replaced by three dots in man output is problematic. But here it doesn't matter, so I changed it to unicode.